.circleci/config/@config.yml

```diff
@@ -1,52 +1,6 @@
 ---
 version: 2.1
 
-references:
```

Comment:
Note to reviewers: Deleted code in this file has mostly been migrated to separate files nested under the `.circleci/config/ directory`. This change was made to make generating the build-release config file easier.